doc/source/rllib/package_ref/policy.rst

```diff
@@ -0,0 +1,49 @@
+.. _policy-reference-docs:
+
+Policy APIs
+===========
+
+.. toctree::
+   :maxdepth: 1
+
+   policy/policy.rst
+   policy/tf_policies.rst
+   policy/torch_policy.rst
+
+The ``Policy`` class contains functionality to compute actions for decision making
+in an environment, as well as computing loss(es) and gradients, updating a neural
+network model as well as postprocessing a collected environment trajectory.
+One or more ``Policy`` objects sit inside a `RolloutWorker's`` "policy_map" and
```

Comment:
We should make sure that `RolloutWorker` is linked to the actual docstring, right?

Reply:
Is RolloutWorker a concept that we want to expose to the user?

Reply:
Yes, I would say that the RolloutWorker API is definitely something the user should be able to sub-class and use in custom Trainers. Both ES and ARS use custom workers.

Reply:
done